.github/workflows/docs.yml

```
@@ -1,29 +1,37 @@
 name: Deploy Sphinx documentation to Pages
 
-# Runs on pushes targeting the default branch
 on:
   push:
-    branches: [main]
-# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+    branches: [main, ghpages]
```

```suggestion
    branches: [main]
```
change before merging

---

changed